### PR TITLE
non zero withdrawal vault

### PR DESCRIPTION
**Pull request by bulbozaur**
Non zero withrawal vault

### CODE REVIEW

File: contracts/0.4.24/Lido.sol [hunk 1]
```diff
@@ -511,8 +519,17 @@ contract Lido is StETHPermit, AragonApp {
     * @dev withdrawal vault address is encoded as a last 160 bits of withdrawal credentials type 0x01
     * @return address of the vault or address(0) if the vault is not set
     */
-    function getWithdrawalVault() external view returns (address) {
-        return _getWithdrawalVault();
+    function getWithdrawalVault() public view returns (address) {
+        return WITHDRAWAL_VAULT_POSITION.getStorageAddress();
+    }
+
+    function setWithdrawalVault(address _withdrawalVault) external {
```
Comment (TheDZhon) [inline contracts/0.4.24/Lido.sol:526]: maybe set all the contracts with `setProtocolContracts` to save bytecode size?

Comment (bulbozaur) [reply contracts/0.4.24/Lido.sol:526]: then there will be conflict in initialize where _initialize_v2 and _setProtocolContracts called

Comment (TheDZhon) [reply contracts/0.4.24/Lido.sol:526]: Didn't get it (we can read old contracts and pass them again for initialization purposes or am I missing smth?)

Comment (folkyatina) [reply contracts/0.4.24/Lido.sol:526]: I see. We can likely merge  initialized_v2 and setProtocolContracts together. Just pass already set contracts in finalize_v2.

Comment (bulbozaur) [reply contracts/0.4.24/Lido.sol]: or just let double writing int initialize method, as far as it won't be called never on already deployed contract, but will be replaced with next upgrade anyway. removing of 2 setters (setWithdrawalVault and setWithdrawaQueue) and concentrating it in setProtocolContracts saves 0.25kb of bytecode as well)

**Pull request closed without merging**